tools/static-assets/server/esm-loader.mjs

```
@@ -0,0 +1,264 @@
+// Loads Meteor packages via standard ESM dynamic imports instead of
+// vm.runInThisContext, eliminating the need for Reify and Module.prototype
+// patching.
+//
+// Generated by `meteor build --format=esm`. Not used by the legacy format.
+//
+// Usage:
+//   import { boot } from './esm-loader.mjs';
+//   await boot('/path/to/bundle/programs/server');
+
+import { createRequire } from 'node:module';
+import { AsyncLocalStorage } from 'node:async_hooks';
```

ESLint

[error] `node:async_hooks` import should occur before import of `node:module`

(import/order)